pepys_admin/maintenance/utils.py

```diff
@@ -69,3 +76,72 @@ def get_str_for_field(value):
         return f"{value:~.2fP}"
     else:
         return str(value)
+
+
+def create_time_filter_dict() -> dict:
+    """Creates a dictionary that has the filter strings as keys, and related datetimes as values."""
+    today = datetime.today()
+    today_as_date = today.date()
+    one_day_delta = timedelta(hours=24)
+    time_filter_dict = {
+        "In past 24 hours": (today - one_day_delta, today),
```

Comment:
@BarisSari - could you store these strings a constants?  Then we can be sure the UI & the backend are expecting the same string.

Reply:
Sure, @IanMayo 👍 I'm updating the code

Reply:
Done: https://github.com/debrief/pepys-import/pull/804/commits/f0bca1867bc9fd93547cfab9ca22ca74520d2d42